### PR TITLE
[octomap_server] Fix updating of color without byte shift

### PR DESCRIPTION
**Pull request by wkentaro**

### CODE REVIEW

File: octomap_server/include/octomap_server/OctomapServer.h [hunk 1]
```diff
@@ -66,7 +66,7 @@
 #include <octomap/octomap.h>
 #include <octomap/OcTreeKey.h>
 
-//#define COLOR_OCTOMAP_SERVER // turned off here, turned on identical ColorOctomapServer.h - easier maintenance, only maintain OctomapServer and then copy and paste to ColorOctomapServer and change define. There are prettier ways to do this, but this works for now
+#define COLOR_OCTOMAP_SERVER
```
Comment (wxmerkt) [inline octomap_server/include/octomap_server/OctomapServer.h]: Can you please deactivate this define? We do not want to change default behaviour.

**Review by wxmerkt — CHANGES_REQUESTED** [review]
Thank you very much for the improvement to get the colour without the byte shift! Can you please revert the change to the `#DEFINE` - very happy to merge this pull request then :)

**Comment by wxmerkt** [comment]
Thank you @wkentaro! 

I have just changed the base branch to `kinetic-devel` and it appears there are conflicts. Could you please have a look and resolve? [We are now developing against kinetic-devel by default]

**Comment by wkentaro** [comment]
Fixed.

**Review by wxmerkt — APPROVED** [review]
Thank you! Will merge once it passes CI.

**Pull request merged**